Content.Shared/CCVar/CCVars.cs

```diff
@@ -341,14 +341,34 @@ public sealed class CCVars : CVars
         public static readonly CVarDef<bool> DebugCoordinatesAdminOnly =
             CVarDef.Create("game.debug_coordinates_admin_only", true, CVar.SERVER | CVar.REPLICATED);
 
+
+        /// <summary>
+        ///     Whether to allow characters to select traits.
+        /// </summary>
+        public static readonly CVarDef<bool> GameTraitsEnabled =
+            CVarDef.Create("game.traits_enabled", true, CVar.REPLICATED);
+
+        /// <summary>
+        ///     How many traits a character can have at most.
+        /// </summary>
+        public static readonly CVarDef<int> GameTraitsMax =
+            CVarDef.Create("game.traits_max", Int32.MaxValue, CVar.REPLICATED);
```

Comment:
It's really late; sorry if I'm staring at it and missing it but where is the value for how many traits can be selected maximum?

Reply:
`Int32.MaxValue`, a practically infinite number.

Reply:
so if I wanted to set it to 5 traits maximum; how would I change this?